docs/pages/web/chartgraph.js

```diff
@@ -0,0 +1,599 @@
+            sandpackExample={<SandpackExample code={biaxial} name="Biaxial" layout="column" />}
+          />
+        </MainSection.Subsection>
+
+        <MainSection.Subsection
+          title="Title & description"
+          description={`A title for the graph in case it’s not displayed elsewhere on the screen. An optional description is available if more context is needed.
+
+Requires props:  \`title\` and \`description"\`.
+`}
+        >
+          <MainSection.Card
+            sandpackExample={<SandpackExample code={title} name="Title & description" />}
+          />
+        </MainSection.Subsection>
+
+        <MainSection.Subsection
+          title="Tooltip"
+          columns={2}
+          description={`For showing more precise details on hover.
+
+Requires props:  \`renderTooltip\`
+          `}
+        >
+          <MainSection.Card
+            cardSize="lg"
+            title="Default tooltip"
+            sandpackExample={<SandpackExample code={tooltip} name="Default tooltip" />}
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="lg"
+            title="No tooltip"
+            sandpackExample={<SandpackExample code={noTooltip} name="No tooltip" />}
+          />
+          <MainSection.Card
+            cardSize="lg"
+            title="Custom tooltip"
+            sandpackExample={<SandpackExample code={customTooltip} name="Custom tooltip" />}
+          />
+        </MainSection.Subsection>
+
+        <MainSection.Subsection
+          title="Legend"
+          description={`Graphs that show more than one category should include a legend to clarify what color or pattern belongs to which category. For single axis charts, a legend isn’t needed when [TagData](/web/tagdata) or [TileData](/web/tiledata) are being used. Legend positions automatically on each layout for easier comprehension.
+
+Requires props: \`legend\`
+          `}
+        >
+          <MainSection.Card
+            cardSize="lg"
+            sandpackExample={<SandpackExample code={legend} name="Legend" />}
+          />
+        </MainSection.Subsection>
+
+        <MainSection.Subsection
+          title="ReferenceArea"
+          description={`Use to highlight an area in a graph for extra context. A common example is showing when data isn’t available.
+
+Requires props: \`referenceAreas\`
+          `}
+        >
+          <MainSection.Card
+            sandpackExample={<SandpackExample code={referenceArea} name="ReferenceArea" />}
+          />
+        </MainSection.Subsection>
+
+        <MainSection.Subsection
+          title="Precision in line graphs"
+          description={`To show exact and accurate data, lines should be rectilinear. When showing trends, forecasts and imprecise data, then lines should be curved to denote that these are just approximations.
+
+
+Requires props: \`elements\` and setting \`precision='estimate'\`
```

Comment:
Maybe props should be like `elements=[{type: "line"  precision: "estimate"}]`

Also just wondering here, since the example is wonky to work with. Will this work on a bar graph? How about multiple lines? Multiple bars